storage/remote/queue_manager.go

```diff
@@ -143,6 +143,33 @@ var (
 		},
 		[]string{queue},
 	)
+	maxNumShards = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "max_shards",
```

Comment:
shards_max is the usual naming

Reply:
`min_shards` and `desired_shards` has to be changed accordingly.

Reply:
didn't know that, thanks!